src/coreclr/jit/llvm.cpp

```diff
@@ -537,15 +537,14 @@ bool canStoreLocalOnLlvmStack(LclVarDsc* varDsc)
     return !varDsc->HasGCPtr();
 }
 
-static bool canStoreArgOnLlvmStack(Compiler::Info &info, CorInfoType corInfoType, CORINFO_CLASS_HANDLE classHnd)
+static bool canStoreArgOnLlvmStack(Compiler* compiler, CorInfoType corInfoType, CORINFO_CLASS_HANDLE classHnd)
 {
     // structs with no GC pointers can go on LLVM stack.
     if (corInfoType == CorInfoType::CORINFO_TYPE_VALUECLASS)
     {
         // Use getClassAtribs over typGetObjLayout because EETypePtr has CORINFO_FLG_GENERIC_TYPE_VARIABLE? which fails with typGetObjLayout
```

Comment:
```suggestion
```
(Presumably)

Reply:
Yes, it doesn't seem to be a problem anymore.  Thanks